### DISCUSSION TITLE
Will rel support auto detecting schema changes and generating migration files

### DISCUSSION THREAD

**Discussion by NoSugarCoffee** [go-rel/rel#337] [General]
It seems that current migration tool chain only support generating migration files by coding. 
Will it support features like [django-admin-makemigrations](https://docs.djangoproject.com/en/4.2/ref/django-admin/#django-admin-makemigrations) or [gorm auto migration](https://gorm.io/docs/migration.html#Auto-Migration) .

**Comment by Fs02** [comment]
this is something I want to do, but I don't have enough bandwidth to do this

**Reply by NoSugarCoffee** [reply]
I really like your design in rel and I also want to contribute something if i can.
May be out of my ability :worried:

**Comment by fairking** [comment]
I don't think it would be useful to me. You might be able to generate something very obvious, but when it comes to real project, you always do that manually, as it's not straight forward. It also comes with data migrations as well. I would not worry about that.
